frontend/locales/en.json

```diff
@@ -630,5 +623,253 @@
         "help": "The primary user interface language"
       }
     }
+  },
+  "cheatsheet": {
+    "button": "Open Cheatsheet",
+    "modal":{
+      "title": "Cheatsheet",
+      "headlines": {
+        "description": "Description",
+        "exampleInput": "Example Input",
+        "exampleOutput": "Example Output",
+        "selectTopic": "Select Topic",
+        "readMoreLink": "Read More"
+      },
+      "noSelection": "Select an entry on the left side to show the instructions."
+    },
+    "categories": {
+      "basic": "Basics",
+      "other": "Other",
+      "embedding": "Embedding",
+      "charts": "Charts & Diagrams"
+    },
+    "basics": {
+      "basicFormatting": {
+        "title": "Basic",
+        "description": "These are the basic markdown formatting rules.",
+        "example": "**Bold**\n__Bold__\n\n*Italic*\n_Italic_\n\n++Underline++\n~~Strikethrough~~\n\nSub~script~~\n\nSuper^script^\n\n==Marked=="
+      },
+      "abbreviation": {
+        "title": "Abbreviation",
+        "description": "Abbreviation definitions create tooltips for matching words. They can be placed defined in the document.",
+        "example": "*[HTML]: Hyper Text Markup Language\n*[W3C]:  World Wide Web Consortium\nThe HTML specification\nis maintained by the W3C."
+      },
+      "footnote": {
+        "title": "Footnotes",
+        "description": "Footnotes can be used to add extra information at the bottom of the page. They can be defined anywhere in the document.",
+        "example": "Here is a footnote reference,[^1] and another.[^longnote]\n\n[^1]: Here is the footnote.\n\n[^longnote]: Here's one with multiple blocks. Subsequent paragraphs are indented to show that they belong to the previous footnote."
+      },
+      "headlines": {
+        "title": "Headlines",
+        "hashtag": {
+          "title": "Hashtag",
+          "description": "Headlines can be used to structure your document into sections. Every headline creates a linkable anchor.",
+          "example": "# Headline 1\n\n## Headline 2\n\n### Headline 3\n\n#### Headline 4\n\n##### Headline 5\n\n###### Headline 6"
+        },
+        "equal": {
+          "title": "Equal",
+          "description": "An alternative form to define the first headline is to append a line of only equal-signs.",
```

Comment:
A second level headline can be defined by using normal dashes instead of equal-signs.

Maybe this section should be named "Alternative" instead of "Equal"

![image](https://user-images.githubusercontent.com/52606896/225650627-2e544bbb-4cde-4e18-877a-4362eda13117.png)